lib/Table/TableColorMap.ts

```diff
@@ -300,6 +313,10 @@ export default class TableColorMap {
     return Color.fromCssColorString(this.colorTraits.regionColor);
   }
 
+  @computed get isDiverging() {
```

Comment:
Maybe good to add a comment on what we mean by diverging - i.e values cross zero.

Reply:
Done! 
I also realised that I needed to add a test to make sure the colorPalette hasn't been set to something that is not diverging - so I have done that